core/src/main/java/org/phoenixframework/liveview/lib/Attribute.kt

```diff
@@ -0,0 +1,50 @@
+package org.phoenixframework.liveview.lib
+
+class Attribute(private var nativeObject: Long) {
+
+    private var _name: String? = null
+
+    /** The name of an attribute */
+    val name
+        get() = _name ?: get_name(nativeObject).also { _name = it }
+    private var _namespace: String? = null
+
+    /** The namespace of an attribute */
+    val namespace
+        get() = _namespace ?: get_namespace(nativeObject).also { _namespace = it }
+
+    private var _val: String? = null
+    val value
+        get() = _val ?: get_value(nativeObject).also { _val = it }
```

Comment:
You can use lazy initialization for this

```suggestion
    val value by lazy {
        get_value(nativeObject)
     }
```